web/styles/compose.css

```diff
@@ -402,6 +409,16 @@
     color: var(--color-text-default);
 }
 
+#compose-textarea.over_limit,
```

Comment:
Is it necessary to use the ID selectors here? Note also that we have SCSS capabilities in the codebase, you you could double-up on these without having to duplicate the `.over_limit` and `.approaching_limit` selectors manually (again, it would be good to hyphenate those, with `over_limit` being corrected in a prep commit).